packages/jest-runtime/src/script_transformer.js

```diff
@@ -33,7 +33,7 @@ export type Options = {|
   collectCoverage: boolean,
   collectCoverageFrom: Array<Glob>,
   collectCoverageOnlyFrom: ?{[key: string]: boolean, __proto__: null},
-  isCoreModule?: boolean,
+  isNativeModule?: boolean,
```

Comment:
why change the name? "native module" in my mind is native addons (https://nodejs.org/api/addons.html), not node core modules.

I suppose it's not public API, but still

Reply:
We already use a `isCoreModule` in `packages/jest-resolve/src/index.js`, which does not quite match the meaning of `isNativeModule`. When I noticed it I decided to rename all the new additions.